source/user-manual/capabilities/file-integrity/fim-fields-rule-mapping.rst

```diff
@@ -0,0 +1,404 @@
+.. Copyright (C) 2021 Wazuh, Inc.
+
+FIM fields rule mapping
+=======================
+
+This guide aims to help to understand how FIM fields can be mapped into rules.
+
+
+FIM - Alerts : fields correspondence
```

Comment:
```suggestion
FIM - Alerts: fields correspondence
```

Reply:
Done in [commit](https://github.com/wazuh/wazuh-documentation/pull/3913/commits/1096a0103d3f508c9dde14df124066b17050626c)